.eslintrc

```diff
@@ -0,0 +1,4 @@
+extends: fxa/server
+
+rules:
+  handle-callback-err: 0
```

Comment:
Do you actually need this rule? The only place I see an error to handle is the callback for `server.register` in `tests/index.js`, and it handles the error by calling `console.error`. Am I missing another?